backend/coins/btc/util/util_test.go

```diff
@@ -60,6 +61,16 @@ func TestPkScriptFromAddress(t *testing.T) {
 	require.Equal(t,
 		[]byte("\x00\x20\x4a\xf2\xe4\x54\x9a\x5c\xbb\x73\x6e\x77\xce\xf5\x2f\xe3\x0b\x9d\xf8\x12\x1d\x73\x56\xab\x20\x05\x46\x3e\xcb\x08\x97\x23\x45\x8d"),
 		pkScript)
+
+	// Taproot: test vector #1 from https://github.com/bitcoin/bips/blob/d7cc20992724ea484087c07d2ed53fb3bc3a108b/bip-0086.mediawiki#test-vectors
+	pubkey := []byte("\xa6\x08\x69\xf0\xdb\xcf\x1d\xc6\x59\xc9\xce\xcb\xaf\x80\x50\x13\x5e\xa9\xe8\xcd\xc4\x87\x05\x3f\x1d\xc6\x88\x09\x49\xdc\x68\x4c")
+	address, err = btcutil.NewAddressTaproot(pubkey, net)
+	require.NoError(t, err)
+	pkScript, err = PkScriptFromAddress(address)
+	require.NoError(t, err)
+	require.Equal(t,
+		[]byte("\x51\x20\xa6\x08\x69\xf0\xdb\xcf\x1d\xc6\x59\xc9\xce\xcb\xaf\x80\x50\x13\x5e\xa9\xe8\xcd\xc4\x87\x05\x3f\x1d\xc6\x88\x09\x49\xdc\x68\x4c"),
```

Comment:
would've been so much easier to compare with 
![image](https://user-images.githubusercontent.com/25405/146942809-3c7792f1-2815-41fa-b205-2ef6b4c2fe28.png)

if it were typed as a regular string and then `hex.DecodeString`-decoded for require.Equal